spacy/pipeline/_parser_internals/ner.pyx

```diff
@@ -385,9 +387,9 @@ cdef class Begin:
         elif st.B_(1).ent_iob == 3:
             # If the next word is B, we can't B now
             return False
-        elif st.B_(1).sent_start == 1:
-            # Don't allow entities to extend across sentence boundaries
-            return False
+        #elif st.B_(1).sent_start == 1:
+        #    # Don't allow entities to extend across sentence boundaries
+        #    return False
```

Comment:
Can be removed?

Reply:
I think these need to be re-instated. So, I did so :). Thanks!